blob/azureblob/azureblob.go

```diff
@@ -15,6 +15,11 @@
 // Package azureblob provides a blob implementation that uses Azure Storage’s
 // BlockBlob. Use OpenBucket to construct a *blob.Bucket.
 //
+// NOTE: SignedURLs for PUT created with this package are not fully portable;
```

Comment:
The result of calling `SignedURL` is a string URL; the will likely send it to a user somehow so that the user can issue the `PUT`. When they do so, they will need to make sure that that `PUT` request has this header in it. I don't think there's anything we can do in code.